src/Common/TTLCachePolicy.h

```diff
@@ -38,12 +38,12 @@ class PerUserTTLCachePolicyUserQuota : public ICachePolicyUserQuota
     bool approveWrite(const UUID & user_id, size_t entry_size_in_bytes) const override
     {
         auto it_actual = actual.find(user_id);
-        Resources actual_for_user{.size_in_bytes = 0, .num_items = 0}; /// assume zero actual resource consumption is user isn't found
+        Resources actual_for_user{.size_in_bytes = 0, .num_items = 0}; /// default if no user is found is no resource consumption
```

Comment:
The comment sounds odd, some commas are needed or maybe this way?
```suggestion
        Resources actual_for_user{.size_in_bytes = 0, .num_items = 0}; /// default, no user -> no resource consumption
```

Reply:
Rewrote the comment, should be more readable now. Thanks.